kahuna/public/js/analytics/track.js

```diff
@@ -103,13 +103,15 @@ track.run(['$rootScope', '$window', 'mixpanelToken', 'mixpanel', 'trackingServic
 
 }]);
 
-track.directive('grTrackClick', ['track', function(track) {
+track.directive('grTrackClick', ['$parse', 'track', function($parse, track) {
 
     return {
         restrict: 'A',
         link: function(scope, element, attrs) {
             const name = attrs.grTrackClick;
-            element.on('click', () => track.action(name));
+            const data = $parse(attrs.grTrackClickData)(scope);
```

Comment:
Out of curiosity, is there a difference between that and `scope.$eval(attrs.grTrackClickData)`?